.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es2020": true,
+    "jasmine": true
+  },
+  "extends": [
+    "closure-es6",
+    "plugin:jasmine/recommended",
+    "google"
+  ],
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
```

Comment:
I don't expect that we'll be using JSX features in this project.

Reply:
Oops, I copied a lot from the previous project as you can tell. Removed it.